packages/ui/src/components/va-badge/VaBadge.demo.vue

```diff
@@ -23,32 +23,22 @@
       </tr>
       <tr>
         <td>
-          <VbCard title="Floated badges">
-            <va-badge text="Regular">
-              Float at
-            </va-badge><br><br>
-            <va-badge
-              text="Bottom"
-              bottom
-            >
-              Float at
-            </va-badge><br><br>
-            <va-badge
-              text="Left"
-              left
-            >
-              Float at
-            </va-badge><br><br>
-            <va-badge
-              text="Bottom+Left"
-              bottom
-              left
-            >
-              Float at
-            </va-badge><br><br>
-            <va-badge text="icon">
-              <va-icon name="face" />
-            </va-badge><br><br>
+          <VbCard title="Transparent">
```

Comment:
Isn't would be better to control transparency by CSS variables?

Reply:
Imo not convenient for dynamic states.